### DISCUSSION TITLE
Optimized conversion options -- drawing to PDF bitmap

### DISCUSSION THREAD

**Discussion by Firestone83** [ImageMagick/ImageMagick#4565] [Digital Image Processing]
Hello to the community!
I would like to discuss about optimized conversion options from source vector PDF with many elements to PDF as a bitmap. My goal is a choose right options to get a fine output with optimized size with quality that leads to read details from the PDF.
Any tips please how to solve it with convert module?
Here is the input example: [_vektor.pdf](https://github.com/ImageMagick/ImageMagick/files/7676478/_vektor.pdf)
It's possible to get output reach to the quality of this output?
[Situace_rastr.pdf](https://github.com/ImageMagick/ImageMagick/files/7676533/Situace_rastr.pdf)
Many thanks for possible feedback.

**Comment by snibgo** [comment]
What version of IM, on what platform?

Is Situace_rastr.pdf a bitmap file? It looks like a vector to me. If it is a bitmap, then what are the dimensions? A suitable "-density" setting will give the same dimensions.

**Comment by Firestone83** [comment]
Firstly thanks for your feedback.
For now I'm trying convert.exe @ ImageMagick 7.1.0-17 Q16-HDRI x64 2021-12-04 @ Win10 x64 platform based.
I'm not sure if it's a right choice of distribution?
And about Situace_rastr.pdf when a take a look at it again you are right. It's finally vector but printed from AutoCAD with other PDF printer. You are right, both are vector.

So for now the question is how make it possible to get properly bitmap with useable quality?
